package.json

```
@@ -4,64 +4,50 @@
   "version": "1.0.0",
   "license": "Apache-2.0",
   "scripts": {
-    "start": "if-env NODE_ENV=production && npm run -s serve || npm run -s dev",
-    "build": "preact build --no-prerender",
-    "serve": "preact build --no-prerender && preact serve",
+    "build": "preact build",
+    "serve": "sirv build --port 8080 --cors --single",
     "dev": "preact watch",
     "lint": "eslint src",
-    "test": "jest ./tests",
-    "deploy": "gulp deploy",
-    "license": "license-checker --csv --production > licenses.csv"
+    "test": "jest"
   },
   "eslintConfig": {
-    "extends": "eslint-config-synacor"
+    "extends": "preact",
+    "ignorePatterns": [
+      "build/"
+    ]
   },
-  "eslintIgnore": [
-    "build/*"
-  ],
   "devDependencies": {
-    "eslint": "^4.9.0",
-    "eslint-config-synacor": "^2.0.2",
-    "gulp": "^3.9.1",
-    "gulp-gh-pages": "^0.5.4",
+    "enzyme": "^3.10.0",
+    "enzyme-adapter-preact-pure": "^2.0.0",
+    "eslint": "^6.0.1",
+    "eslint-config-preact": "^1.1.0",
     "identity-obj-proxy": "^3.0.0",
```

I think we probably still want the license-checker.
```suggestion
    "identity-obj-proxy": "^3.0.0",
    "license-checker": "^20.2.0",
```